electrode_models/nucleation_single_particle/initialize.py

```diff
@@ -106,9 +106,10 @@ class electrode:
     electrode.SVptr = {}
     electrode.SVptr['phi_ed'] = np.array([0])
     electrode.SVptr['phi_dl'] = np.array([1])
-    electrode.SVptr['C_k_ed'] = np.arange(2, 2 + electrode.bulk_obj.n_species)
-    electrode.SVptr['C_k_elyte'] = np.arange(2 + electrode.bulk_obj.n_species, 
-        2 + electrode.bulk_obj.n_species + electrode.elyte_obj.n_species)
+    electrode.SVptr['Area'] = np.arange([3])
```

Comment:
Is the area independent from the current state of `Histogram`?  It seems like, if I know `Histogram` at any point in time/space, I can also calculate the local surface area...

Reply:
Its used in the previous model, and I thought there was a reason why but now I can't remember...